doc/api/http.md

```diff
@@ -1003,11 +1003,11 @@ added: v0.1.29
 * `callback` {Function}
 * Returns: {boolean}
 
-Sends a chunk of the body. By calling this method
-many times, a request body can be sent to a
-server. In that case, it is suggested to use the
-`['Transfer-Encoding', 'chunked']` header line when
-creating the request.
+Sends a chunk of the body. This method can be called multiple times. If no
+`Content-Length` is set, data will automatically be encoded in HTTP Chunked
+transfer encoding, so that server knows when the data ends. The
+`Transfer-Encoding: chunked` header will be added. Call `end()` when finish
+sending data so that node would send the trailer.
```

Comment:
I think the last sentence should be rephrased (does `node` here refers to Node.js? what is a trailer in this context?), and the reference to `end()` should be a link to its documentation.

```suggestion
`Transfer-Encoding: chunked` header will be added. Calling [`request.end()`][]
is necessary to finish sending the request.
```